devimint/src/lib.rs

```diff
@@ -260,6 +304,14 @@ impl Lnd {
         .await?;
         Ok(())
     }
+
+    pub async fn kill(&mut self) -> Result<()> {
```

Comment:
kill should take `self` because a `Lnd` value shouldn't exist without a running lnd